src/mgmt/srv_target.c

```diff
@@ -899,11 +816,26 @@ ds_mgmt_tgt_map_update_pre_forward(crt_rpc_t *rpc, void *arg)
 		if (servers[i].se_flags & SERVER_IN) {
 			if (existing)
 				continue;
-			rc = add_server(group, &servers[i]);
+			D_DEBUG(DB_MGMT, "add rank=%u uri=%s nctxs=%u\n",
+				servers[i].se_rank, servers[i].se_uri,
+				servers[i].se_nctxs);
```

Comment:
unless you use it elsewhere, seems you can now get rid of se_nctxs

Reply:
I thought we might need the number of contexts in the future. If not, I'll remove it in a future patch. Thanks for the reminder, Alex.